File: My Leet Code Java Solutions/Solution88.java
//88. Merge Sorted Array

import java.util.*;

class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int counter=0;
        for(int i=m; i<m+n; i++){ // m = 3, m+n = 6
            nums1[i]=nums2[counter];
            counter++;
        }
        Arrays.sort(nums1);
    }
}

File: 2022-2023/MyLeetCodeSolutions/PythonSolutions/Solution88.py
#88. Merge Sorted Array
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        counter = 0
        for i in range(m, m+n):
            nums1[i] = nums2[counter]
            counter += 1
        nums1.sort()
